vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -301,6 +310,13 @@ func verifyRemoveSubnetValidatorTx(
 		return nil, false, err
 	}
 
+	if backend.Config.UpgradeConfig.IsEActivated(currentTimestamp) {
```

Comment:
This `E` check isn't strictly required since there are no `SetSubnetManager` calls until `E` is activate. However, doing a timestamp check avoids a DB read that is performed in `GetSubnetManager`.